website/default_config.py

```diff
@@ -10,6 +10,11 @@
     'v1': '1.0',
     'v2': '2.0',
 }
+PICARD_VERSIONS = {
+    'stable': ('1.4.2', 'http://picard.musicbrainz.org/'),
+    'beta': ('2.0.0.beta1', 'https://github.com/metabrainz/picard/releases/tag/2.0.0.dev4'),
+    'dev': ('2.0.0.dev4', 'https://github.com/metabrainz/picard/releases/tag/2.0.0.dev4'),
```

Comment:
Can't we retrieve those infos directly from git + tags ?
https://github.com/metabrainz/picard/releases lists "latest release" (=stable), "pre-release" (=beta)

Reply:
It looks like all the information is there, so I'm pretty sure it could be scraped from the page.  We might even be able to get the latest dev version because they are shown without either the "latest release" or "pre-release" tags.